bellman/src/domain/linear.rs

```diff
@@ -0,0 +1,163 @@
+use std::ops::{Add, Sub};
+use ff::{ScalarEngine, Field};
+
+/// Represents a variable in our constraint system.
```

Comment:
Vague, a variable means many things, please add documentation to specify

Reply:
Agreed, this is their naming choice that I haven’t changed yet.

Reply:
Coefficient perhaps?

Reply:
I’ll find out whether coefficient is an appropriate name based on its usage in gadgets.

Reply:
Actioned, thanks